### DISCUSSION TITLE
Host on website

### DISCUSSION THREAD

**Discussion by thedeepakgaur** [devlikeapro/waha#337] [Q&A] [answered]
Is there any way to create a website to provide api to businesses

**Comment by allburov** [comment, marked as answer]
Yes, you need to get WAHA Pro and start creating website
https://waha.devlike.pro/pricing/

For instance, you can use white labeled swagger there
https://waha.devlike.pro/docs/how-to/swagger/#white-label

There's no way to white label Dashboard, but we'll do it if PRO patreon
would ask!

https://waha.devlike.pro/docs/how-to/waha-dashboard/

вс, 19 мая 2024 г., 22:47 thdeepakrajput2 ***@***.***>:

> Is there any way to create a website to provide api to businesses
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/devlikeapro/whatsapp-http-api/discussions/337>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABBMR3Q6ICDES4YH5CL5FZDZDDCRJAVCNFSM6AAAAABH6O5XXWVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGY4TGMRVGI>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
